website/pages/en/index.js

```diff
@@ -414,51 +369,67 @@ class Index extends React.Component {
-                <h2>
-                  <translate>Watch Talks about Jest</translate>
-                </h2>
+                <MarkdownBlock>
+                  <translate>
+                    The Jest core team and contributors regularly speak about
+                    [Jest and Delightful JavaScript
+                    Testing](https://www.youtube.com/watch?v=cAKYQpTC7MA). Check
+                    out our talk about [Building High-Quality JavaScript
+                    Tools](https://www.youtube.com/watch?v=PvabBs_utr8) at
+                    jsconf.eu 2017 and our talk about [Jest as a
+                    Platform](https://www.youtube.com/watch?v=NtjyeojAOBs) at
+                    ReactiveConf 2017.
+                  </translate>
+                </MarkdownBlock>
+              </div>
+              <div className="blockContent">
                 <div>
+                  Cupiditate repellendus sit rerum est minus nam. Velit placeat
+                  reiciendis placeat cumque. Sint et voluptatibus neque ad
+                  dolore sunt. Esse est odio possimus nulla. Ipsam voluptate
+                  maiores quo sed maiores excepturi. Quisquam sed sint incidunt
+                  placeat est.
+                </div>
+              </div>
+            </div>
+          </Container> */}
+          <div className="container section-container imageAlignSide lightBackground twoByGridBlock">
+            <div className="wrapper">
+              <div className="gridBlock">
+                <div className="blockContent rotate-right">
+                  <h2>
+                    <translate>Open Collective</translate>
+                  </h2>
                   <MarkdownBlock>
                     <translate>
-                      The Jest core team and contributors regularly speak about
-                      [Jest and Delightful JavaScript
-                      Testing](https://www.youtube.com/watch?v=cAKYQpTC7MA).
-                      Check out our talk about [Building High-Quality JavaScript
-                      Tools](https://www.youtube.com/watch?v=PvabBs_utr8) at
-                      jsconf.eu 2017 and our talk about [Jest as a
-                      Platform](https://www.youtube.com/watch?v=NtjyeojAOBs) at
-                      ReactiveConf 2017.
+                      With so many users, the core team of Jest uses an [Open
+                      Collective](https://opencollective.com/jest) for
+                      non-Facebook contributors.
                     </translate>
                   </MarkdownBlock>
+                  <Contributors />
+                </div>
+                <div className="blockContent rotate-left">
+                  <h2>
+                    <translate>Who uses Jest?</translate>
+                  </h2>
+                  <MarkdownBlock>
+                    <translate>
+                      A lot of people! With
+                      [8.5m](https://www.npmjs.com/package/jest) downloads in
```

Comment:
any way of keeping these numbers updated automatically?

Might be a cool docusaurus feature (showing stats from npm and github), /cc @endiliey

Reply:
Yep, that would be really useful